modules/Miner.js

```diff
@@ -0,0 +1,136 @@
+const BN = require('bn.js');
```

Comment:
Based on whole file there's only one description what it does: `Solve PoW task`. A small description about the algorithm would be nice.